docs/book/src/faq.md

```diff
@@ -0,0 +1,88 @@
+
+# Kubebuilder FAQ
```

Comment:
WDYT is about to be the only FAQ? Is it not a little prolixly add Kubebuilde here?